tox.ini

```diff
@@ -6,9 +6,9 @@ passenv = TEMPEST_CONFIG_DIR
 setenv =
   OS_TEST_PATH={toxinidir}/f5lbaasdriver/test/tempest/tests
 deps =
-  -rrequirements.test.txt
+  pytest
 changedir = f5lbaasdriver/test/tempest/tests/
-commands = py.test -vra {posargs}
+commands = py.test {posargs}
```

Comment:
I remember this now..
Some flags are in 2 places.   See `scripts/run_neutron_lbaas.sh`
I'm going to push _all_ flags into `systest/scripts/run_neutron_lbaas.sh` for finer grained control.